### PR TITLE
Copy data selections fixings

### PR DESCRIPTION
**Pull request by elizzhao**
Added some more edits to the work done in PR #928 per Kate's comments. 

-Changes to language 
-Remove the user's ability to choose a different dest workspace if they're heading down one of the conflict paths

### CODE REVIEW

File: src/components/ExportDataModal.js [hunk 1]
```diff
@@ -96,8 +96,8 @@ export default _.flow(
         content: `WARNING: ${runningSubmissionsCount} workflows are currently running in this workspace. ` +
           'Copying the following data could cause failures if a workflow is using this data.'
       }),
-      requiredFormLabel('Destination'),
-      h(WorkspaceSelector, {
+      !((hardConflicts.length !== 0) || moreToDelete || (softConflicts.length !== 0)) && requiredFormLabel('Destination'),
+      !((hardConflicts.length !== 0) || moreToDelete || (softConflicts.length !== 0)) && h(WorkspaceSelector, {
```
Comment (panentheos) [inline src/components/ExportDataModal.js]: We could eliminate the duplicate logic here by wrapping these elements in a `Fragment`

**Review by panentheos — APPROVED** [review]
Looks good.

File: src/components/ExportDataModal.js [hunk 2]
```diff
@@ -97,23 +98,25 @@ export default _.flow(
         content: `WARNING: ${runningSubmissionsCount} workflows are currently running in this workspace. ` +
           'Copying the following data could cause failures if a workflow is using this data.'
       }),
-      requiredFormLabel('Destination'),
-      h(WorkspaceSelector, {
-        workspaces: _.filter(Utils.isValidWsExportTarget(workspace), workspaces),
-        value: selectedWorkspaceId,
-        onChange: v => this.setState({ selectedWorkspaceId: v })
-      }),
+      !((hardConflicts.length !== 0) || moreToDelete || (softConflicts.length !== 0)) && h(Fragment, [
```
Comment (zarsky-broad) [inline src/components/ExportDataModal.js:101]: For the `.length === 0`'s, you could actually just do `foo.length` or `!!foo.length`

**Review by zarsky-broad — APPROVED** [review]

**Review by katevoss — APPROVED** [review]
This looks good! One strange thing I noticed is that when I copied a few pieces of information back and forth between my 2 workspaces, that when I tried to copy one data entry from one workspace to another, that the other workspace was no longer listed in the dropdown. It is also true on dev so it's not specific to this branch. Do you have an idea about what might be going on here? If not I'll make a ticket. 

screen shot 2018-12-07 at 5 08 59 pm

**Comment by elizzhao** [comment]
Not really sure what's going on @katevoss there. It probably has something to do with how we do workspace selection. It likely requires a little more investigation to see if it's a data specific problem or a workspace copying problem in general. I'm going to go ahead and merge this in since it's been sitting out for a while, but we should file that bug!

**Pull request merged**